cmd/kured/main.go

```diff
@@ -122,6 +124,11 @@ func rebootRequired() bool {
 func rebootBlocked() bool {
 	if prometheusURL != "" {
 		alertNames, err := alerts.PrometheusActiveAlerts(prometheusURL, alertFilter)
+		weekday := time.Now().Weekday()
+		if noWeekendReboot && (int(weekday) == 0 || int(weekday) == 6) {
+			log.Warnf("Reboot blocked: day of week: %v",weekday)
```

Comment:
seems more "info" than "warning"

Reply:
Thanks for the relevant comment, I have updated the log to info